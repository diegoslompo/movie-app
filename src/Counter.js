import React, { Component } from 'react'

class Counter extends Component {
  render () {
    return (
      <div className="counter">
        <h2> vocÃª tem {this.props.count * 2} counters.</h2>
      </div>
    )
  }
} 

export default Counter;