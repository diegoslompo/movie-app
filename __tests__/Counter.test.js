import React from "react"
import ShallowRenderer from 'react-test-renderer/shallow';
import Counter from '../src/Counter'

// descreve o teste no componente
describe('Testing component Counter', () => { 
    it('should render corretly', () => {

        const renderer = new ShallowRenderer()
        const wrapper = renderer.render(
            <Counter count={6} />
        )

        expect(wrapper).toMatchSnapshot();

    })
})