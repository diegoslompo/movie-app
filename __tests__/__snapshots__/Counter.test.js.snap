// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component Counter should render corretly 1`] = `
<div
  className="counter"
>
  <h2>
     vocÃª tem 
    12
     counters.
  </h2>
</div>
`;
